src/main/java/ua/mateacademy/tania/lesson5/taskfruits/FruitKind.java

```diff
@@ -0,0 +1,22 @@
+package ua.mateacademy.tania.lesson5.taskfruits;
+
+/**
+ * Created by Tania Nebesna on 04.01.2019.
+ */
+public enum FruitKind{
+    APPLE("grenni"),
+    PEAR("bera"),
+    ORANGE(""),
+    BANANA(""),
+    GRAPES("kysh-mysh");
+
+    FruitKind(String name) {
+        this.name = name;
```

Comment:
You newer use the fields of enum. I think task was not only create them

Reply:
it`s true, I never use field, I will change it